spec.yaml

```diff
@@ -135,12 +339,21 @@ components:
           type: string
     Entity:
```

Comment:
Sliter framleis litt med `Entity`. Er `Resource` , som i [dcat:Resource](https://www.w3.org/TR/vocab-dcat-2/#Class:Resource) betre?

Uansett: Vi kan godt skyve på denne.

Reply:
Her er noen tanker:
- `Resource` kunne fungert like godt
- `Resource` ville da kollidere med Jena sin `Resource` class

Reply:
Aiaiai, kan du ringe Jena og be de endre?

Neida, ser den. Alternativer:

1. Entitiy (as-is), eller
2. dcatResource (eller liknande)